DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/bridging/test_bridging_jumbo_frame.py

```diff
@@ -0,0 +1,382 @@
+            ]}])
+    assert out[0][device_host_name]["rc"] == 0, f"Verify that FDB static entries added.\n{out}"
+
+    address_map = (
+        #swp port, tg port,     tg ip,     gw,        plen
+        (ports[0], tg_ports[0], "1.1.1.2", "1.1.1.1", 24),
+        (ports[1], tg_ports[1], "2.2.2.2", "2.2.2.1", 24),
+        (ports[2], tg_ports[2], "3.3.3.2", "3.3.3.1", 24),
+        (ports[3], tg_ports[3], "4.4.4.2", "4.4.4.1", 24),
+    )
+
+    dev_groups = tgen_utils_dev_groups_from_config(
+        {"ixp": port, "ip": ip, "gw": gw, "plen": plen}
+        for _, port, ip, gw, plen in address_map
+    )
+
+    await tgen_utils_traffic_generator_connect(tgen_dev, tg_ports, ports, dev_groups)
+    
+    list_macs = ["aa:bb:cc:dd:ee:11", "aa:bb:cc:dd:ee:12",
+                 "aa:bb:cc:dd:ee:13", "aa:bb:cc:dd:ee:14"]
+
+    streams = {
+        f"bridge_{dst + 1}": {
+            "ip_source": dev_groups[tg_ports[src]][0]["name"],
+            "ip_destination": dev_groups[tg_ports[dst]][0]["name"],
+            "srcMac": list_macs[src],
+            "dstMac": list_macs[dst],
+            "type": "raw",
+            "protocol": "802.1Q",
+        } for src, dst in ((3, 0), (2, 1), (1, 2), (0, 3))
+    }
+
+    await tgen_utils_setup_streams(tgen_dev, config_file_name=None, streams=streams)
+
+    await tgen_utils_start_traffic(tgen_dev)
+    await asyncio.sleep(traffic_duration)
+    await tgen_utils_stop_traffic(tgen_dev)
+
+    # check the traffic stats
+    stats = await tgen_utils_get_traffic_stats(tgen_dev, "Traffic Item Statistics")
+    for row in stats.Rows:
+        assert float(row["Loss %"]) == 0.000, f'Failed>Loss percent: {row["Loss %"]}'
+        assert tgen_utils_get_loss(row) == 0.000, \
+        f"Verify that traffic from {row['Tx Port']} to {row['Rx Port']} forwarded.\n{out}"
+    
+    out = await BridgeFdb.show(input_data=[{device_host_name: [{"options": "-j"}]}],
+                         parse_output=True)
+    assert out[0][device_host_name]["rc"] == 0, "Failed to get fdb entry.\n"
+
+    fdb_entries = out[0][device_host_name]["parsed_output"]
+    learned_macs = [en["mac"] for en in fdb_entries if "mac" in en]
+    for mac in list_macs:
+        err_msg = f"Verify that source macs have been learned.\n"
+        assert mac in learned_macs, err_msg
+
+    await tgen_utils_stop_protocols(tgen_dev)
+
+
+@pytest.mark.asyncio
+async def test_bridging_frame_min(testbed):
+    """
+    Test Name: test_bridging_frame_min
+    Test Suite: suite_functional_bridging
+    Test Overview: Verifying that Jambo frames of min size are learned and forwarded.
+    Test Author: Kostiantyn Stavruk
+    Test Procedure:
+    1.  Init bridge entity br0.
+    2.  Set ports swp1, swp2, swp3, swp4 master br0.
+    3.  Set bridge br0 admin state UP.
+    4.  Set entities swp1, swp2, swp3, swp4 UP state.
+    5.  Set min jumbo frame MTU size support on ports swp1, swp2, swp3, swp4.
+    6.  Set ports swp1, swp2, swp3, swp4 learning ON.
```

Comment:
See step 9. Those are conflicting steps. To fix it read carefully your test purpose.